move/bayarea/sources/collection.move

```diff
@@ -22,7 +22,7 @@ module addr::bayarea_collection {
 
     /// The account that is allowed to create the collection. For now we just enforce
     /// that the collection creator is the same account that published the module.
-    const PERMITTED_COLLECTION_CREATOR: address = @addr;
+    const PERMITTED_COLLECTION_CREATOR: address = @creator;
```

Comment:
Why make this change? I think it's confusing because the contract for now assumes the deployer is also the collection creator.

Reply:
We should use object code deployment to deploy contracts. And the module address `addr` is the object address which is dynamically created on each deploy, therefore we can't use `addr`. It's better to define who the creator is explicitly.

Reply:
Good call, sounds good. Note that we've already deployed to mainnet though.

Reply:
That's fine :)